gdc_client/download/client.py

```diff
@@ -47,19 +47,19 @@ def download_small_groups(self, smalls):
                         filename = os.path.join(self.directory, filename.split('=')[1])
                     else:
                         filename = time.strftime("gdc-client-%Y%m%d-%H%M%S")
-                    log.info('Saving grouping {0}/{1}'.format(i+1, groupings_len))
+                    log.info('Saving grouping {}/{}'.format(i+1, groupings_len))
```

Comment:
Yeah I thought about that today, too. I'll probably put it all back and then some.